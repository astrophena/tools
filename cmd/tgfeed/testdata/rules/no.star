# Â© 2024 Ilya Mateyko. All rights reserved.
# Use of this source code is governed by the ISC
# license that can be found in the LICENSE.md file.

feed(
    url="https://example.com/feed.xml",
)
