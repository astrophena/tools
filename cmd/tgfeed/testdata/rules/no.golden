[
  {
    "chat_id": "test",
    "parse_mode": "HTML",
    "reply_markup": {
      "inline_keyboard": [
        []
      ]
    },
    "text": "ðŸ”— \u003ca href=\"https://example.com/block\"\u003eBlock this item\u003c/a\u003e #examplecom"
  },
  {
    "chat_id": "test",
    "parse_mode": "HTML",
    "reply_markup": {
      "inline_keyboard": [
        []
      ]
    },
    "text": "ðŸ”— \u003ca href=\"https://example.com/extensions\"\u003eItem with extensions\u003c/a\u003e #examplecom"
  },
  {
    "chat_id": "test",
    "parse_mode": "HTML",
    "reply_markup": {
      "inline_keyboard": [
        []
      ]
    },
    "text": "ðŸ”— \u003ca href=\"https://example.com/keep\"\u003eKeep this item\u003c/a\u003e #examplecom"
  }
]